src/components/dialogs/dataSources/COVIDcast.svelte

```diff
@@ -47,7 +47,11 @@
   });
 
   export function importDataSet() {
-    return importCOVIDcast({ data_source, signal, geo_type, geo_value });
+    return fetchCOVIDcastMeta().then((res) => {
+      const meta = res.filter((row) => row.data_source === data_source && row.signal === signal);
+      const time_type = meta[0].time_type || 'day';
```

Comment:
If this doesnt have a value (which should be unlikely if not impossible), should we let it fall through instead of forcing `'day'`?  I think that insisting on this default is presumptuous, especially if/when we add other time granularities.
```suggestion
      const time_type = meta[0].time_type;
```